protected/humhub/modules/user/views/account/changeUsername.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use yii\widgets\ActiveForm;
+use \humhub\compat\CHtml;
+
+$userModule = Yii::$app->getModule('user');
+?>
+
+<?php $this->beginContent('@user/views/account/_userProfileLayout.php') ?>
+    <div class="help-block">
+         <?php echo Yii::t('UserModule.views_account_changeUsername', 'Your current username is <b>{username}</b>. You can change your current username here. Changing the username can make some links unusable, for example old links to the profile', ['username' => CHtml::encode(Yii::$app->user->getIdentity()->username)]); ?>
```

Comment:
Please change text to: _Note: Changing your username will invalidate old profile links (not including mentionings)_ and wrap this (only this sentence) within a <div class="alert alert-warning">

Reply:
@buddh4  what tag should wrap this sentence?

Reply:
Sorry, got lost: `<div class="alert alert-warning">`